src/test/java/org/tuberlin/de/server/controller/DownloadJavaControllerTest.java

```diff
@@ -0,0 +1,58 @@
+package org.tuberlin.de.server.controller;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+
+import static org.easymock.EasyMock.expect;
+import static org.easymock.EasyMock.anyObject;
+import static org.powermock.api.easymock.PowerMock.*;
+
+/**
+ * Unit test for the {@link DownloadJavaController} class that controls that a file
+ * is written into the response
+ */
+
+import javax.servlet.ServletOutputStream;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Created by Fabian on 10.12.2015.
+ */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ HttpServletRequest.class, HttpServletResponse.class, DownloadJavaController.class})
+public class DownloadJavaControllerTest {
```

Comment:
Ok, I did not know that. But I think we will add tests in the future (maybe even integration tests), and then it would be convenient to start them separately, if possible. 
If not, just add one line saying how and when they are being executed - for clarity.